packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -261,14 +266,19 @@ UserSettingsMenuBody.displayName = 'UserSettingsMenuBody';
 
 const UserSettingsMenu = (props: Props) => {
   const intl = useIntl();
+  const menuLabelId = useFieldId(undefined, menuLabelSequentialId);
 
   return (
     <div data-test="user-settings-menu">
-      <Downshift stateReducer={stateReducer}>
+      <AccessibleHidden>
+        <span id={menuLabelId}>
+          <FormattedMessage {...messages.menuLabel} />
+        </span>
+      </AccessibleHidden>
+      <Downshift stateReducer={stateReducer} labelId={menuLabelId}>
         {(downshiftProps) => (
           <div>
             <button
-              role="user-menu-toggle"
```

Comment:
This is an invalid role so I removed it.
I also searched this and MC frontend repositories and didn't find any test using this role name.